Sources/Sentry/SentryClient.m

```diff
@@ -366,6 +368,17 @@ - (SentryEvent *_Nullable)prepareEvent:(SentryEvent *)event
         event = self.options.beforeSend(event);
     }
 
+    // The callback is also called when SentrySDK.crashedLastRun is true and the
+    // SentryCrashReportSink didn't capture the crash event yet and the user manually captures an
+    // event with level fatal.
+    // We could also call this callback in SentryHub.captureCrashEvent, but then the event would
+    // miss detail from SentryClient.prepareEvent.
+    if (nil != self.options.onCrashedLastRun && SentrySDK.crashedLastRun
+        && !SentrySDK.crashedLastRunCalled && event.level == kSentryLevelFatal) {
+        SentrySDK.crashedLastRunCalled = YES;
```

Comment:
M: is this a public API?
what's about getting rid of it and just setting the `onCrashedLastRun` callback to null?

Reply:
probably not a public API because its added into `SentrySDK+Private.h` so ignore me

Reply:
No, it is not a public API, but good pointing it out.